docs/developer-guide/testing.md

```diff
@@ -79,6 +79,6 @@ However, as a word of caution, be aware that existing integration tests will
 start using the new programs immediately, so this should only be done if the
 modified program is backward compatible.
 
-## Kubernetes Integration Tests
+## Kubernetes Operator Unit and Integration Tests
 
-Detailed decription TBD
+See [Testing locally](./develop-operator.md#testing-locally) for the Kubernetes operator.
```

Comment:
I think it would make more sense to move the content (as is) from develop-operator.md to here and have develop-operator.md hyperlink to this section. This is a testing doc, so the content should be here.

Reply:
I considered doing that, but was trying to avoid too many changes.  However, I agree that would be better and will move it.